content/en/docs/installation/compatibility.md

```diff
@@ -40,9 +40,13 @@ To address this, the webhook can be run in the host network so it can be reached
 by cert-manager, by setting the `webhook.hostNetwork` key to true on your
 deployment, or, if using Helm, configuring it in your `values.yaml` file.
 
-Note that since kubelet uses port `10250` by default on the host network, the
-`webhook.securePort` value must be changed to a different, free port.
+**IMPORTANT**: kubelet on AWS uses port `10250` by default on the host network, which
+clashes with the default port for the cert-manager webhook.
```

Comment:
suggestion: use Docsy's "warning" shortcode:

```suggestion
{{% alert title="Important" color="warning" %}}

kubelet on AWS uses port `10250` by default on the host network, which
clashes with the default port for the cert-manager webhook.

{{% /alert %}}
```

It looks like this:

![warning-docsy](https://user-images.githubusercontent.com/2195781/119668538-8f349a00-be37-11eb-8588-a1e0f9d7bf57.png)